tests/server/endpoints/test_delivery_message_endpoint.py

```diff
@@ -33,3 +37,30 @@ def test_get_delivery_message_matching_order(
 
     # THEN the response contains a message
     assert response.json["message"]
+
+
+@pytest.mark.parametrize(
+    "analysis_status, expected_status_code",
+    [
+        (AnalysisStatus.COMPLETED, HTTPStatus.OK),
+        (AnalysisStatus.RUNNING, HTTPStatus.PRECONDITION_FAILED),
+    ],
+)
+def test_get_delivery_message_order(
+    client: FlaskClient,
+    server_case: Case,
+    server_case_in_same_order: Case,
```

Comment:
are these used at all below? 👴

Reply:
Yes, but testing Flask is very weird 🙃 There must be better ways to fill the database, but specifying these imports do the trick as well. So basically this ensures that the order and the cases are in the database.